src/sickle.ts

```diff
@@ -146,12 +168,14 @@ class Annotator {
    * Returns empty string if there is no existing annotation.
    */
   private existingClosureAnnotation(p: ts.PropertyDeclaration | ts.ParameterDeclaration) {
-    let comments = ts.getLeadingCommentRanges(this.file.text, p.pos);
+    let text = p.getFullText();
+    let comments = ts.getLeadingCommentRanges(text, 0);
```

Comment:
I noticed there are functions with names like getJSDoc in the TypeScript source but for some reason they're not in the public API.  Might be a version thing.